node/impl/full/eth.go

```diff
@@ -88,8 +90,6 @@ type EthEventAPI interface {
 var (
 	_ EthModuleAPI = *new(api.FullNode)
 	_ EthEventAPI  = *new(api.FullNode)
-
-	_ EthModuleAPI = *new(api.Gateway)
```

Comment:
We should maintain this property, we can do so by implementing the new methods on the `EthAPI` directly as Magik said.

Reply:
Ah, ok, fair enough.